src/components/NewCourse.js

```diff
@@ -20,6 +22,8 @@ class NewCourse extends React.Component {
     this.state = {
       name: '',
       shortcode: '',
+      isUnlisted: false,
```

Comment:
Thought - this could be a good opportunity to make a few improvements to this component to match what `NewQueue` does - see https://github.com/illinois/queue/blob/master/src/components/NewQueue.js. Notably:

* Use hooks for simpler state management.
* Use `onSubmit` on the `<Form>` to use native browser handling of form submits instead of manually handling the `Enter` keypress.

This could be outside the scope of this PR though - up to you if you want to make these improvements now or save them for later.